examples/qml_minimal/qml/main.qml

```diff
@@ -18,6 +18,7 @@ Window {
     title: qsTr("Hello World")
     visible: true
     width: 640
+    color: "white"
```

Comment:
As discussed, this only happens when building with cargo. Is some env var missing or it not linking to something?